surface/include/pcl/surface/3rdparty/poisson4/ppolynomial.hpp

```diff
@@ -29,6 +29,7 @@ DAMAGE.
 #include "factor.h"
 
 #include <cstdio>
+#include <cstdlib>
```

Comment:
Maybe add a comment to why its required, else it might be removed by one of the Include-what-you-use tool that remove "unncessary includes" ?

Reply:
```suggestion
#include <cstdlib>  // for malloc, needed by gcc-5
```